protoc-cli/src/main/java/io/ballerina/protoc/builder/stub/Field.java

```diff
@@ -137,7 +137,7 @@ public Field build() {
 
         private Builder(DescriptorProtos.FieldDescriptorProto fieldDescriptor, String fieldType) {
             this.fieldDescriptor = fieldDescriptor;
-            this.type = fieldType;
+            this.type = fieldType != null ? fieldType : "empty:Empty";
```

Comment:
Since field type is a java string, do we need to assign the `empty:Empty` type when for empty strings as well?

Reply:
Here, the field type is null when we define it as `google.protobuf.Empty` in the proto file. The corresponding Ballerina is in the protobuf module[1]. We are assigning the ballerina type here so it generates the correct type.

1. https://github.com/ballerina-platform/module-ballerina-protobuf/blob/main/ballerina/modules/types.empty/empty.bal#L18